Neos.Flow/Classes/Mvc/View/JsonView.php

```diff
@@ -200,7 +195,10 @@ public function setConfiguration(array $configuration)
      */
     public function render()
     {
-        $this->controllerContext->getResponse()->setContentType('application/json');
+        /** @var ActionMessages $actionMessages */
```

Comment:
the json view should probably just return a new Response instead of that.

... then only fusion uses the ActionResponse ...